spec/requests/v0/legacy_form1095_bs_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
```

Comment:
I moved the original specs here and renamed them to make later deletion very simple. The only changes I made were to add the flipper flag and to remove tests for a statsd log I removed (no longer needed).